Update README.md (#4278)
xournalpp/xournalpp: README.md

[![Build Status](https://dev.azure.com/xournalpp/xournalpp/_apis/build/status/CI?branchName=master)](https://dev.azure.com/xournalpp/xournalpp/_build/latest?definitionId=1&branchName=master)
[![Join the chat at https://gitter.im/xournalpp/xournalpp](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/xournalpp/xournalpp?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)

<img src="readme/main.png" width=550px% title="Xournal++ on GNU/Linux"/>

## Translations

Would you like to see Xournal++ in your own language? Translators are welcome to contribute to Xournal++.

> Translation resources are in the following locations:
> * [Crowdin](https://crowdin.com/project/xournalpp/)
> * [desktop/com.github.xournalpp.xournalpp.appdata.xml](desktop/com.github.xournalpp.xournalpp.appdata.xml)
> * [desktop/com.github.xournalpp.xournalpp.desktop](desktop/com.github.xournalpp.xournalpp.desktop)
> * [desktop/com.github.xournalpp.xournalpp.xml](desktop/com.github.xournalpp.xournalpp.xml)

Reviewer: I think a slightly more explicit version reads better:
```suggestion
You can contribute translations to the following locations:
* [Crowdin](https://crowdin.com/project/xournalpp/)
* Submit a pull request on GitHub to change:
  * [desktop/com.github.xournalpp.xournalpp.appdata.xml](desktop/com.github.xournalpp.xournalpp.appdata.xml) (
  * [desktop/com.github.xournalpp.xournalpp.desktop](desktop/com.github.xournalpp.xournalpp.desktop)
  * [desktop/com.github.xournalpp.xournalpp.xml](desktop/com.github.xournalpp.xournalpp.xml)
```

<tr>
<td>

## GNU/Linux

<img src="readme/main.png" width=100% title="Xournal++ on GNU/Linux"/>

</td><td>

## Windows 10

<img src="readme/main-win.png" width=100% title="Xournal++ on Windows 10"/>

</td></tr><tr><td>

## macOS Catalina

<img src="readme/main-mac.png" width=100% title="Xournal++ on macOS Catalina"/>

</td><td>

## Xournal++ Mobile on Chromium OS

<img src="https://gitlab.com/TheOneWithTheBraid/xournalpp_mobile/-/raw/master/fastlane/metadata/android/en_US/images/tenInchScreenshots/03.png" width=100% title="Xournal++ Mobile on Chromium OS"/>

</td></tr><tr><td>

## Toolbar / Page Background / Layer

Multiple page background, easy selectable on the toolbar

> <img src="readme/background.png" width=100% title="Xournal++"/>

Reviewer: ```suggestion
<img src="readme/background.png" width=100% title="Changing the page background in Xournal++"/>
```

</td><td>

## Layer sidebar and advanced layer selection

> <img src="readme/layer.png" width=100% title="Xournal++"/>

Reviewer: ```suggestion
<img src="readme/layer.png" width=100% title="Layers in Xournal++"/>
```

</td></tr><tr><td>

## Multiple predefined and fully customizable toolbars

> <img src="readme/toolbar.png" width=100% title="Xournal++"/>

Reviewer: ```suggestion
<img src="readme/toolbar.png" width=100% title="Custom toolbars in Xournal++"/>
```
